src/core/tests/Test_CoreMain.py

```diff
@@ -906,6 +906,88 @@ def test_auto_assessment_success_with_installation_in_prev_operation_on_same_seq
 
         runtime.stop()
 
+    def test_auto_assessment_resets_maintenance_run_id_and_health_store_id_to_None(self):
+        # operation #1: Installation
+        argument_composer = ArgumentComposer()
+        argument_composer.operation = Constants.INSTALLATION
+        argument_composer.maintenance_run_id = "8/27/2021 02:00:00 PM +00:00"
+        argument_composer.health_store_id = "8/28/2021 02:00:00 PM +00:00"
+        argument_composer.classifications_to_include = ["Security", "Critical"]
+        runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.APT)
+        runtime.set_legacy_test_type("SuccessInstallPath")
+        CoreMain(argument_composer.get_composed_arguments())
+        # check telemetry events
+        self.__check_telemetry_events(runtime)
+        # check status file
+        with runtime.env_layer.file_system.open(runtime.execution_config.status_file_path, 'r') as file_handle:
+            status_file_data = json.load(file_handle)[0]["status"]
+        self.assertTrue(status_file_data["operation"] == Constants.INSTALLATION)
+        substatus_file_data = status_file_data["substatus"]
+        self.assertEqual(len(substatus_file_data), 4)
+        self.assertTrue(substatus_file_data[0]["name"] == Constants.PATCH_ASSESSMENT_SUMMARY)
+        self.assertTrue(substatus_file_data[0]["status"].lower() == Constants.STATUS_SUCCESS.lower())
+        # check started by set to 'User'
+        self.assertTrue(json.loads(substatus_file_data[0]["formattedMessage"]["message"])['startedBy'], Constants.PatchAssessmentSummaryStartedBy.USER)
```

Comment:
please use assertEquals if we are comparing two objects, this way if test failed it output the expected value and actual value instead of true or false

Reply:
assertTrue is being used in other similar checks all throughout our test suite. This was a conscious decision made long back simce AssertEquals() is deprecated in Python 3x. The choice was between using assertEqual vs assertTrue back then